packages/contracts/package.json

```diff
@@ -47,6 +47,7 @@
     "testnet": "truffle develop --testnet",
     "prettier": "prettier --write ./**/*.{ts,js,sol}",
     "testblock:generate": "node utils/testblock-generator",
+    "deploy:local": "hardhat run scripts/deploy.ts --network localhost",
```

Comment:
The network name is a trigger to run `register` method for the test coordinator in the deploy script.
May we can change `testnet` instead `localhost` for this.

Reply:
Network `localhost` is a reserved name for hardhat. If we change to `testnet`, we need to add `testnet` section in `hardhat.config.ts`. Is it expected? I think it's more straightforward now if our intention is to deploy on local node. No strong opinion. If you think `testnet` is better, I'll change it.

Reply:
I just saw your pr, https://github.com/wanseob/zkopru/pull/14/files. I'll modify it.